### ISSUE TITLE
Send email report  not working on Android 13 and Gmail

### ISSUE THREAD

**Issue by arcanewater** [ACRA/acra#1179]
**Describe the bug**
Looks like something has changed again with latest Gmail (or Android) update as it stopped working with sending emails, previously #1118 fixed it but now it stopped working again, i unable to get any usefull logs as it tries to open Gmail but it closes immediately. There is a shortcut for temp fix if you have multiple emails and it pops selection menu you choose gmail it works okay, but if you only have Gmail its not working

**Version**
 - Android: 13
 - ACRA: 5.9.7

**Comment by itanium99** [comment]
However, this cannot be the case in general. On a Samsung Tablet device, Android 13, OneUI 5.1 and ACRA 5.9.7, Gmail is started automatically, but only if I have only one e-mail app installed. As soon as I install Outlook in addition, Outlook is always started. So I never see the selector because the method sendWithSelector brings up the exception ActivityNotFoundException, which then falls back to sendLegacy. I added a separate routine to enable the user to send the error log by Mail afterwards, which is working because this is called in my app without previous crash. I know this all won't help you but i wanted to express there are obviously different behaviors on different devices.

**Comment by arcanewater** [comment]
Interesting is that this was already working but now again failing (Pixel 6)

**Comment by F43nd1r** [comment]
Please test with 5.9.8-beta01 - I have removed using selector intents on tiramisu. Need data on how this changes things for you.

**Comment by arcanewater** [comment]
Tried upgrading to 5.9.8-beta01 but now i receive error where i am extending CrashReportDialog

```
error: cannot access CrashReportDialog
public final class CustomReportDialog extends org.acra.dialog.CrashReportDialog {
                                                             ^
  bad class file: \.gradle\caches\transforms-3\fd18062a6c38c3bd05dcff78b783d9bd\transformed\acra-dialog-5.9.8-beta01-api.jar(/org/acra/dialog/CrashReportDialog.class)
    class file has wrong version 61.0, should be 55.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

**Comment by F43nd1r** [comment]
@arcanewater Unintended side effect of 15d73799fa5db6b030a7f6caf1f69230e4784ed9 is release requires Java 17. I'll see if I can make the next release compatible with 11 again.

**Comment by arcanewater** [comment]
Email with this change is working again 👍

**Comment by yuriykulikov** [comment]
Works now on my Pixels. Thank you!

**Comment by arcanewater** [comment]
I see there are already changes for JDK 11 support possible for new release @F43nd1r ?

**Comment by itanium99** [comment]
(1) With 5.9.8-beta02 selector is showing up after my app has crashed - providing GMail and Outlook (OK)
(2) But there is no attachment (txt) in the mail generated afterwards (NOK)
(3) After uninstalling Outlook, GMail is started automatically without selector (OK) and attachment is correctly there (OK)

**Comment by itanium99** [comment]
Problem (2) still existing under ACRA 5.10.0
Log just shows "No email client supporting attachments found. Attachments will be ignored" when showing up selector for GMail and Outlook.

**Comment by F43nd1r** [comment]
@itanium99 in the future please open new issues instead of responding to closed ones - I have a lot of traffic in my inbox and can easily miss comments on closed stuff (as I did with your first message).

**Comment by itanium99** [comment]
I apologize for the confusion. Since I was involved here the day after this tick was created, I thought my feedback belonged in here. After all, except for a heart emoji, no other response came. Thank you for your efforts. I will continue to test intensively.

**Comment by F43nd1r** [comment]
No worries, as I caught your message this time your scenario (2) is now fixed with `5.10.1`. Thanks for testing.

**Comment by itanium99** [comment]
I tried my scenarios with 5.10.1 - everything is working now as expected. Thanks a lot!
